### DISCUSSION TITLE
What triggers a change to the updated_at field?

### DISCUSSION THREAD

**Discussion by mpfc75** [trakt/trakt-api#263] [Q&A] [answered]
I am trying to understand what causes the "updated_at" field to be changed/updated. I thought I could use this field to determine if something has changed since I last checked it however the field seems to be updated far more often than I would have imagined. 

For example, the show "**America's Funniest Home Videos**" (trakt=317794, slug=america-s-funniest-home-videos), Season 1, Episode 1 was "first_aired" = "1990-01-15T00:00:00.000Z", yet "updated_at" = "2021-02-11T22:43:29.000Z".

It seems odd to me that an episode that was aired over 30 years ago would be updated less than 24 hours ago. I thought it might be votes or comments, but the vote count is 3 and the comment count is zero.

**Comment by rectifyer** [comment, marked as answer]
Pretty much any data will update that timestamp, mostly likely any stats (play count, etc) is updating it.

**Reply by michaldrabik** [reply]
Will show's **updated_at** be updated if its season/episodes change? I mean for example, there is a new episode added or removed from one of show's seasons (for any reason) and it's the only change?

**Reply by MaxHasADHD** [reply]
I would like to know this as well. I'm guessing it does not, as `updated_at` is referring to when the data of the object changed rather than something its connected to, and there isn't a number of episodes property (only aired episodes), so you would need to fetch the seasons for a show and check if the season has changed. I'm trying to figure this out now so I can fetch the least amount of information while being accurate and performant. For long running shows like game shows, I only need to fetch the latest and newest seasons for instance instead of 20+ seasons.

**Reply by tysonkerridge** [reply]
These may help based on my understanding:
https://github.com/trakt/api-help/issues/17#issuecomment-457409322
https://github.com/trakt/api-help/issues/217#issuecomment-1276221741
